src/ImageDeleteModal.jsx

```diff
@@ -5,13 +5,21 @@ import cockpit from 'cockpit';
 
 const _ = cockpit.gettext;
 
+function sortTags(a, b) {
+    if (a.endsWith(":latest"))
+        return -1;
+    if (b.endsWith(":latest"))
+        return 1;
+    return a > b;
```

Comment:
This looks wrong. The compare function is supposed to return -1, 0, or 1, not false/true -- turning the latter into a number in Python just  yields NaN. How about `a.localeCompare(b)`?

Reply:
fixed using `localCompare`